src/endpoints/http/collection.js

```diff
@@ -0,0 +1,25 @@
+function message(message) {
+  return { message: message };
+}
+
+/**
+ * Add the route to create a new collection for the users App.
+ * This route expects a db connection to be present on the request object.
+ *
+ * @param {object} server - The express server.
+ */
+export function createCollection(server) {
+  server.post("/collection", (req, res) => {
+    if (!req.body.name) {
+      return res.status(400).send(message('name is required'));
+    }
+
+    req.db.createCollection(req.body.name, err => {
```

Comment:
There are some restrictions like name length but these are errors reported by fh-db which will be passed to `next`. I didn't see any reason to recheck the name here.
What do you think?